docs/user/cli/console.rst

```diff
@@ -0,0 +1,99 @@
+GeoMesa Scala Console
+=====================
+
+The GeoMesa tools have a ``scala-console`` command that starts a Scala REPL configured
+for use with GeoMesa. The command will place the GeoMesa classpath and configuration
+for that distribution on the REPL's classpath. Additionally it will preload commonly
+used imports. The command also comes with tooling that will provide the option to
+download and run the appropriate version Scala if it is not available. This is a
+portable install and will not alter the machines current configuration.
+
+Example usage
+-------------
+
+The following example was performed against the FileSystem Datastore generated by the
+:ref:`fsds_quickstart`. It shows how to use the ``scala-console`` command to connect
+to a FileSystem Datastore, discover the feature type names, get the schema and query for data.
+
```

Comment:
Should probably add a `.. note::` here to emphasize that this is running against the output of the quick start

Reply:
Done